### PR TITLE
added 2 user-defined tags, and image field

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
This will currently make an empty tag if tag 2 isn't supplied.  I think we can get around that though.

File: inst/rmarkdown/templates/hugo/resources/default.md [hunk 1]
```diff
@@ -7,6 +7,9 @@ title: $if(title)$$title$$endif$
 categories: Data Science
 tags: 
   - R
+  - $if(tag1)$$tag1$$endif$
+  - $if(tag2)$$tag2$$endif$
+image: $if(image)$$image$$endif$
```
Comment (ldecicco-USGS) [inline inst/rmarkdown/templates/hugo/resources/default.md]: Even better:

```
$if(image)$image:$endif$ $if(image)$$image$$endif$
```

then do the same for tag1 and tag2, so they only show up if the user adds them.

Comment (wdwatkins) [inline inst/rmarkdown/templates/hugo/resources/default.md]: 👍   Works well.  It is important that tags be last though - if there is an empty line with tag2 not present and image comes after, it breaks Hugo.

**Comment by ldecicco-USGS** [comment]
:+1:

**Comment by ldecicco-USGS** [comment]
Want to tag a release and update the grantools list?

**Comment by wdwatkins** [comment]
Sure.  Can you try this out first though?  I get weird errors about it not finding the png file and trying to read the statsServiceMap.knit.md and statsServiceMap.utf8.md that only happen if Hugo is running when I knit.  Not sure if its a problem.

**Comment by ldecicco-USGS** [comment]
Sure, one sec...
And we'll have to remember to bump up this version before tagging a release.

**Comment by ldecicco-USGS** [comment]
seems good to me. I think it's just because momentarily their are no files to serve.

**Comment by wdwatkins** [comment]
0.0.2 then?

**Comment by ldecicco-USGS** [comment]
yup

**Pull request merged**